### PR TITLE
915 model program unit tests

### PR DESCRIPTION
**Pull request by mohamedmorsy**
@pkdash could you please review?

### CODE REVIEW

**Comment by pkdash** [comment]
@mohamedmorsy This PR needs to first pass Jenkins checks.

**Comment by mohamedmorsy** [comment]
@pkdash this PR is ready now to review. Thanks!

**Comment by pkdash** [comment]
@zhiyuli Can you review this?

File: hs_model_program/tests/test_model_program_metadata.py [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+    def test_extended_metadata_CRUD(self):
+        # test the core metadata at this point
+        # there should be a title element
+        self.assertEquals(self.resModelProgram.metadata.title.value, 'Test Model Program Resource')
+
+        # there should be a creator element
+        self.assertEquals(self.resModelProgram.creator.username, 'user1')
+
+        # # there should be a type element
```
Comment (zhiyuli) [inline hs_model_program/tests/test_model_program_metadata.py]: the comment here matches the code?

File: hs_model_program/tests/test_model_program_metadata.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+        self.assertNotEqual(self.resModelProgram.metadata.type, None)
+
+        # there should be an identifier element
+        self.assertEquals(self.resModelProgram.metadata.identifiers.count(), 1)
+
+        # there should be rights element
+        self.assertNotEqual(self.resModelProgram.metadata.rights, None)
```
Comment (zhiyuli) [inline hs_model_program/tests/test_model_program_metadata.py]: same as above

**Comment by zhiyuli** [comment]
+1

**Comment by pkdash** [comment]
@mohamedmorsy You should merge this.

**Pull request merged**